install-feature fails on Win 10 with "productInfo exited with return code 1"

On version 3.1, the **install-feature** goal is failing on a new MicroProfile Starter project via a `mvn package` command with:

> 
> [INFO] BUILD FAILURE
> [INFO] ------------------------------------------------------------------------
> [INFO] Total time:  07:19 min
> [INFO] Finished at: 2019-10-21T13:33:09-04:00
> [INFO] ------------------------------------------------------------------------
> [ERROR] Failed to execute goal io.openliberty.tools:liberty-maven-plugin:3.1:install-feature (package-server) on project demo: productInfo exited with return code 1 -> [Help 1]

To me it looks the code here: 

```
            if (OSUtil.isWindows()) {
                productInfoFile = installDirectory + "\\bin\\productInfo.bat";
            } else {
                productInfoFile = installDirectory + "/bin/productInfo";
            }
            ProcessBuilder pb = new ProcessBuilder(productInfoFile, action);
```

is not even successfully launching the process.

I added a test to ci.maven and verified that parentheses in the path of the productInfo command does cause the reported problem. 

Here is the error my test case received when running on Windows:

00:25:32] [INFO] [ERROR] Failed to execute goal io.openliberty.tools:liberty-maven-plugin:3.2-SNAPSHOT:install-feature (install-server-features) on project install-features-2-it: productInfo exited with return code 1. The productInfo command run was `C:\projects\ci-maven\liberty-maven-plugin\target\it\install-features(2)-it\target\liberty\wlp\bin\productInfo.bat validate`

The fix is going into ci.common via PR https://github.com/OpenLiberty/ci.common/pull/103.

--- a/liberty-maven-plugin/src/it/install-features(2)-it/pom.xml
+++ b/liberty-maven-plugin/src/it/install-features(2)-it/pom.xml
@@ -1,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.openliberty.tools.it</groupId>
+        <artifactId>tests</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    
+    <artifactId>install-features-2-it</artifactId>
+    <packaging>pom</packaging>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.openliberty.tools</groupId>
+                <artifactId>liberty-maven-plugin</artifactId>
+                <version>@pom.version@</version>
+                <executions>
+                    <execution>
+                        <id>install-liberty-server</id>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>install-server</goal>
+                        </goals>
+                    </execution>
+                    <!-- Create the server -->
+                    <execution>
+                        <id>create-server</id>
+                        <phase>pre-integration-test</phase>
+                        <goals>
+                            <goal>create</goal>
+                        </goals>
+                    </execution>
+                    <!-- Install the features (mongodb-2.0, oauth-2.0 & openid-2.0) using only the server name -->
+                    <execution>
+                        <id>install-server-features</id>
+                        <phase>pre-integration-test</phase>
+                        <goals>
+                            <goal>install-feature</goal>
+                        </goals>
+                        <configuration>
+                            <features>
+                                <acceptLicense>true</acceptLicense>
+                            </features>
+                        </configuration>
+                    </execution>
+                </executions>
+                <configuration>
+                    <serverName>test</serverName>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+</project>

--- a/liberty-maven-plugin/src/it/install-features(2)-it/src/test/resources/server.xml
+++ b/liberty-maven-plugin/src/it/install-features(2)-it/src/test/resources/server.xml
@@ -1,0 +1,9 @@
+<server description="default server">
+    <featureManager>    
+        <feature>osgiBundle-1.0</feature>
+        <feature>jsp-2.3</feature>
+        <feature>mongodb-2.0</feature>
+        <feature>openid-2.0</feature>
+        <feature>couchdb-1.0</feature>
+    </featureManager>     
+</server>

